client/reader/sidebar/style.scss

```diff
@@ -63,7 +63,7 @@
 
 	.sidebar__menu-link:hover .count {
 		border-color: var(--color-sidebar-menu-hover-text);
-		outline-color: red;
+		outline-color: #f00;
```

Comment:
This isn't currently visible (just the border shows). It's not an error, so hesitant to change to that red variable.